app/dashboard/src/App.tsx

```diff
@@ -356,7 +358,8 @@ function AppRouter(props: AppRouterProps) {
     }
   }, [localStorage, inputBindingsRaw])
 
-  const mainPageUrl = getMainPageUrl()
+  const mainPageUrl = getRootUrl()
+  mainPageUrl.pathname += '/drive'
```

Comment:
Why do we need `/drive` here now when we didn't before?

Reply:
great question... i think it's no longer required because the Dashboard handles redirecting to `/drive` itself so it can safely be changed back

Reply:
umm, it seems like this was the issue...